### DISCUSSION TITLE
How to reference key from Key Store in a playbook?

### DISCUSSION THREAD

**Discussion by LukaszC86** [semaphoreui/semaphore#2099] [Q&A]
Hi,
is there any way to reference key (login and password) from Key Store in a playbook? Currently I had to add them in environment variables, but it would be more secure if I could be able to reference keys from Key Store in a playbooks. Thanks for an answer!

**Comment by CptLuxx** [comment]
I have the same question, i cant find anything in the documentation about this.

**Reply by LukaszC86** [reply]
Some time ago Secrets have been added for Environments. You can get them via lookup('env', 'SECRET_NAME') just like enviroment variables.

![image](https://github.com/user-attachments/assets/4f5f9ad4-c4ca-4a5f-83d9-d1c55a4444f0)
